lightning/src/ln/msgs.rs

```diff
@@ -689,6 +689,9 @@ pub enum ErrorAction {
 	},
 	/// The peer did something harmless that we weren't able to process, just log and ignore
 	IgnoreError,
+	/// The peer did something harmless (and expected) that we weren't able to meaningfully
+	/// process. Simply ignore.
+	IgnoreCommonError,
```

Comment:
What do you think about `HarmlessError/MinorError`, a bit more meaningful w.r.t to IgnoreError ?

Reply:
Hmm, part of what I wanted to capture is that it is "common" in that it appears regularly. Ultimately that's the real difference between `log_trace` and `log_debug` - debug should still be readable, but trace you probably need to grep to get anything useful :). Maybe drop the `Ignore` part and just do `ExpectedError`?

Reply:
Done